wom/src/main/scala/wom/views/GraphPrint.scala

```diff
@@ -46,15 +49,15 @@ final class GraphPrint(executableCallable: ExecutableCallable) {
     val clusterNumber = clusterCounter.getAndIncrement()
     val id = s"cluster_$clusterNumber"
 
-    def handleScatterVariableNode(scatterVariableNode: ScatterVariableNode): NodesAndLinks = {
+    def handleScatterVariableNode(): NodesAndLinks = {
       scatterNode.scatterVariableNodes foldMap { node =>
         val dotNode = DotScatterVariableNode.apply(node, clusterNumber)
         val links = upstreamLinks(node.linkToOuterGraph.graphNode, dotNode, Map.empty)
         NodesAndLinks(Set(dotNode), links)
       }
     }
 
-    val scatterExpressionNodesAndLinks = scatterNode.scatterVariableNodes.foldMap(handleScatterVariableNode)
+    val scatterExpressionNodesAndLinks = scatterNode.scatterVariableNodes.foldMap(_ => handleScatterVariableNode())
```

Comment:
The duplication of the expression `scatterNode.scatterVariableNodes.foldMap` here and on line 53 makes me think that this might be an artifact of a mid-work refactor that didn't quite get finished but that, luckily, doesn't affect the result. All the same, I'm wondering if this can be simplified to just:
```
    val scatterExpressionNodesAndLinks = handleScatterVariableNode()
```

Going even further, I wonder if the original author meant this:
```
    def handleScatterVariableNode(node: ScatterVariableNode): NodesAndLinks = {
      val dotNode = DotScatterVariableNode.apply(node, clusterNumber)
      val links = upstreamLinks(node.linkToOuterGraph.graphNode, dotNode, Map.empty)
      NodesAndLinks(Set(dotNode), links)
    }

    val scatterExpressionNodesAndLinks = scatterNode.scatterVariableNodes.foldMap(handleScatterVariableNode)
```

Reply:
You may be right, according to codecov (which does have false negatives, but I generally trust it...) this code might not have been given tests. So I'm approaching my limits of comfort in refactoring (potentially) untested code.

How about if I revert this change if it's controversial?

Reply:
I agree with @breilly2's version. That looks most likely to be what was originally intended. In terms of tests, I thought we had some `womgraph` generation output tests, I can see if I can find any (maybe we just didn't include scatters in them?)

Reply:
See: https://github.com/broadinstitute/cromwell/blob/develop/womtool/src/test/scala/womtool/graph/WomDotGraphTest.scala